Add Lecture 01: What's money and why it breaks (#1)
edilmedeiros/bitcoin-course: 01-digital-economy.md

space, time, and scale.
A good medium of exchange must facilitate trade across different locations (space), preserve value for future transactions (time), and allow practical exchanges of both small and large amounts (scale).

While IOUs improve barter by introducing an intermediate good, they still rely on the trustworthiness and future actions of specific individuals.
This dependence limits their usefulness, especially across broader communities where personal reputation cannot be easily verified.
A natural solution is to adopt "neutral" goods—objects that are not promises of future action but tangible, transferable items.
Neutral goods reduce the need for trust:
their acceptance does not depend on knowing or trusting the previous holder.
Furthermore, because their value is based on general expectations of future acceptance rather than personal obligation, they become more universally recognizable as mediums of exchange.
Throughout history, societies discovered that natural goods such as shells, stones, cattle, salt, and beads could serve this role.

> Over time, precious metals like gold and silver emerged as especially favored intermediaries, particularly in ancient Greece.

Reviewer: Maybe add that while there was no coinage back in 3500 BC Mesopotamia, silver was used as a UoA. From _Debt: The First 5000 Years_, p39:

> The Sumerian economy was dominated by vast temple and palace
complexes. These were often staffed by thousands: priests and officials,
craftspeople who worked in their industrial workshops, farmers and
shepherds who worked their considerable estates. Even though ancient
Sumer was usually divided into a large number of independent citystates,
by the time the curtain goes up on Mesopotamian civilization
around 3500, temple administrators already appear to have developed
a single, uniform system of accountancy-one that is in some ways
still with us, actually, because it's to the Sumerians that we owe such
things as the dozen or the 24-hour day. **The basic monetary unit was
the silver shekel.** One shekel's weight in silver was established as the
equivalent of one gur, or bushel of barley. A shekel was subdivided
into 60 minas, corresponding to one portion of barley-on the principle 
that there were 30 days in a month, and Temple workers received
two rations of barley every day. It's easy to see that " money" in this
sense is in no way the product of commercial transactions. It was actually
created by bureaucrats in order to keep track of resources and
move things back and forth between departments.

Author: That would be a nice addition, but I'm not trying to be historically precise or comprehensive on the text. Can't see how more details will add to the story that money is a human technology that assumed many forms and to the problems of the current fiat form.

Instead, its usefulness as an economic medium of exchange derives solely from the expectation that others will also accept it in future transactions.
Moreover, its suitability for economic exchanges does not hinge on any alternative practical use—such as being consumed as food or processed in industry.
This fundamental shift—from trusting individuals to trusting the generalized marketability of an object—marked a major milestone in the evolution of money.

Beyond spatial acceptance, a good candidate for money must also perform well over time.
It must allow individuals to accumulate value for future use without significant deterioration or loss.
Imagine someone who wishes to eventually buy a house but can only save small amounts at a time.
If the chosen medium of exchange perishes, rots, or erodes in value over time, saving becomes practically impossible.
Goods like fresh produce or cigarettes, although used as money in extreme circumstances, are poorly suited for long-term value storage.

Durability is crucial, but so is scarcity:
the medium must not be easily reproducible, otherwise saving is undermined by inflation.

> To illustrate, consider an economy with 50 stones used as money and 10 spears available for trade.
> If each spear costs 5 stones on average, and suddenly someone finds a mine producing 50 additional stones, the average price of spears would fall to 2 stones.

Reviewer: If the money supply increases to 100 stones, the the average price for a spear would be 10 stones: _100/10 = 10_.
```suggestion
To illustrate, consider an economy with 50 stones used as money and 10 spears available for trade.
If each spear costs 5 stones on average, and suddenly someone finds a mine producing 50 additional stones, the average price of spears would increase to 10 stones.
```

It is not that spears became more valuable;
rather, the stones lost their purchasing power.
This phenomenon, known as inflation, was historically observed when the influx of gold and silver from the Americas into Europe eroded the value of existing currencies.
Inflation is not neutral:

> those who first receive new money can purchase goods before prices adjust, gaining an unfair advantage.

Reviewer: ```suggestion
those who first receive new money can purchase goods before prices adjust, gaining an unfair advantage—this is the Cantillon Effect.
```

Author: Thanks, added this together with an external reference on 8dd8485.

# Lecture 1: What is Money? Why It Breaks

In this first lecture, our goal is to explore what money is, how it emerged, and how it evolved over time.
We are not aiming for a historically precise narrative, nor for an ultimate philosophical definition.
Rather, we are interested in understanding how money has been abstracted throughout history and how its problems, particularly in its fiat form, set the stage for the emergence of Bitcoin.
Our focus is practical:
by the end of this discussion, we want a clear sense of the problems Bitcoin was designed to address.

## Why Trade Needs Money

Imagine two individuals, Alice and Bob.
Alice is an electronics teacher, offering specialized lessons to those interested in learning about circuits, semiconductors, and signal processing.
Bob, on the other hand, is a farmer who grows carrots.
One day, Alice goes to Bob in a fair and offers an introductory lecture about the problems of money and how Bitcoin addresses them in return for a some carrots. At first glance, the exchange seems promising.
Carrots are nutritious food, and knowledge is a valuable asset.

> Bob has been hearing about Bitcoin quite frequently lately and was interesting in understanding the technology to expand his business.

Reviewer: ```suggestion
Bob has been hearing about Bitcoin quite frequently lately and was interested in understanding the technology to expand his business.
```

Author: Added in 7b701d2.

The same bill might change hands dozens of times before anyone seeks redemption.
Redemption becomes necessary only when the neutrality of gold is essential—for instance, in international trade or when doubts arise about a bank’s solvency.
This widespread acceptance of paper, independent of frequent physical settlement, is what made banking systems so powerful—and, as we will see, also vulnerable to overreach and abuse.

## Digital Messaging as Money

Let’s revisit Alice and Bob.
This time, Alice opens an account at her bank and receives a plastic card linked to it—a credit or debit card.
With this card, Alice can now initiate electronic payments by signing digital messages that instruct her bank to transfer a portion of her balance to Bob's account.
When Alice makes a payment, the bank doesn’t hand over physical cash or gold;
it updates its internal ledger.
It deducts the value from Alice’s account and credits the same amount to Bob’s.
Moments later, Bob’s bank notifies him that the payment has been received and his balance adjusted.

> That's when he knows he can handle the basket of fresh carrots to Alice;

Reviewer: ```suggestion
That's when he knows he can hand the basket of fresh carrots to Alice;
```

Author: Added in 7b701d2.

# Lecture 1: What is Money? Why It Breaks

In this first lecture, our goal is to explore what money is, how it emerged, and how it evolved over time.
We are not aiming for a historically precise narrative, nor for an ultimate philosophical definition.
Rather, we are interested in understanding how money has been abstracted throughout history and how its problems, particularly in its fiat form, set the stage for the emergence of Bitcoin.
Our focus is practical:
by the end of this discussion, we want a clear sense of the problems Bitcoin was designed to address.

## Why Trade Needs Money

Imagine two individuals, Alice and Bob.
Alice is an electronics teacher, offering specialized lessons to those interested in learning about circuits, semiconductors, and signal processing.
Bob, on the other hand, is a farmer who grows carrots.

> One day, Alice goes to Bob in a fair and offers an introductory lecture about the problems of money and how Bitcoin addresses them in return for a some carrots. At first glance, the exchange seems promising.

Reviewer: ```suggestion
One day, Alice goes to Bob at a fair and offers an introductory lecture about the problems of money and how Bitcoin addresses them in return for some carrots. At first glance, the exchange seems promising.
```

Barter relies on the unlikely alignment of immediate desires between two parties—a situation known as the double coincidence of wants.
Alice must find not just anyone, but someone who both needs electronics lectures and has something she personally desires at that moment.
The odds of matching her specialized knowledge to the immediate wants of others are slim.
The burden of matching these wants falls heavily on individuals, forcing them to spend significant time searching for suitable partners.

For Alice, this is particularly problematic.
Teaching is a service dependent on her time and attention.
Every hour spent seeking a trade partner is an hour not spent providing her lectures.
In a barter system, marketing her expertise competes directly with delivering her expertise.
The inefficiency is so profound that Alice might eventually abandon teaching altogether, turning instead to a more commonly needed activity, such as growing vegetables herself.
Society, in this way, does not benefit from specialization:
individuals are pressured to conform to more universally tradable goods and services rather than developing and offering unique talents.

> We would not have the complex information-driven society if not for more powerful economic tool on our belts.

Reviewer: ```suggestion
We would not have the complex information-driven society of today if not for a more powerful economic tool on our belts.
```

For an asset to be considered money, it must become the most marketable intermediate good in the economy—the one asset that individuals universally accept in exchange, confident that they can later trade it for whatever they desire.
As Carl Menger explains in On the Origins of Money, money emerges not by decree but through a gradual process where individuals gravitate toward goods that possess superior qualities of marketability.
We will talk about these qualities next.

## What Makes Good Money

To understand why some goods thrive as mediums of exchange while others fail, we must consider how well they perform across three critical dimensions:
space, time, and scale.
A good medium of exchange must facilitate trade across different locations (space), preserve value for future transactions (time), and allow practical exchanges of both small and large amounts (scale).

While IOUs improve barter by introducing an intermediate good, they still rely on the trustworthiness and future actions of specific individuals.
This dependence limits their usefulness, especially across broader communities where personal reputation cannot be easily verified.

> A natural solution is to adopt "neutral" goods—objects that are not promises of future action but tangible, transferable items.

Reviewer: ```suggestion
A natural solution is to adopt "neutral" goods—objects that are not promises of future action, but rather tangible and transferable items.
```

Imagine someone who wishes to eventually buy a house but can only save small amounts at a time.
If the chosen medium of exchange perishes, rots, or erodes in value over time, saving becomes practically impossible.
Goods like fresh produce or cigarettes, although used as money in extreme circumstances, are poorly suited for long-term value storage.

Durability is crucial, but so is scarcity:
the medium must not be easily reproducible, otherwise saving is undermined by inflation.
To illustrate, consider an economy with 50 stones used as money and 10 spears available for trade.
If each spear costs 5 stones on average, and suddenly someone finds a mine producing 50 additional stones, the average price of spears would increase to 10 stones.
It is not that spears became more valuable;
rather, the stones lost their purchasing power.
This phenomenon, known as inflation, was historically observed when the influx of gold and silver from the Americas into Europe eroded the value of existing currencies.
Inflation is not neutral:
those who first receive new money can purchase goods before prices adjust, gaining an unfair advantage. 
This is called the Cantillon Effect.

> Since governments always tried to control currency, they were always in a privileged position when we talk about inflation.

Reviewer: ```suggestion
Since governments have always tried to control currency, they were always in a privileged position when we talk about inflation.
```

It is something human beings invent—and reinvent—whenever the need to coordinate trade arises.
As we've seen throughout this lecture, money takes form in response to the practical problems of cooperation across time, space, and uncertainty.
Whether through IOUs, beads, gold coins, or digital messages, people will discover ways to create a medium of exchange that suits their needs.
That means anyone, anywhere, exposed to the challenge of economic exchange, can rediscover the logic of money.
This realization—that money is a tool, not a decree—should empower us to think critically about its design, its failures, and its possible futures.

This perspective helps explain why money appears so universally across human societies, even those that never interacted.
It is not a product of any particular culture or authority—it is a recurring solution to a recurring problem.
In that sense, money is part of human nature:
not because it is biologically programmed, but because it emerges wherever humans trade, plan, and specialize.
It is a natural consequence of living in a society where cooperation and temporal deferral of value are essential to progress.

> Wherever people act under scarcity and seek to improve their condition though trading, money—or something like it—will arise.

Reviewer: ```suggestion
Wherever people act under scarcity and seek to improve their condition through trading, money—or something like it—will arise.
```

We will spend the rest of this course exploring how Bitcoin works and how it addresses the problems we’ve outlined.
But we will also remain grounded:
Bitcoin is a superior implementation of money in many respects, but whether it becomes widely used as money is not up to any authority.
That decision rests, as it always has, with individuals making voluntary choices.
Bitcoin is not merely a new asset—it is a technological implementation of money, grounded in formal rules, economic incentives, and decentralized enforcement.
Decentralization of money is also the decentralization of the decision of what to use as money.

It's up to you to decide if Bitcoin is money.
And I hope this course will help you do so.

## References

Lawrence H. White, Better Money: Gold, Fiat, or Bitcoin?, Cambridge University Press, 2023.

F. A. Hayek, The Denationalization of Money: The Argument Refined, Institute of Economic Affairs, 1978.

Carl Menger, On the Origins of Money, 1892.

Ludwig von Mises, The Theory of Money and Credit, 1912.

Murray Rothbard, What Has Government Done to Our Money?, 1963.

Nick Szabo, "Shelling Out: The Origins of Money," 2002.

> (The Cantillon Effect: Because of Inflation, We’re Financing the Financiers)[https://fee.org/articles/the-cantillon-effect-because-of-inflation-we-re-financing-the-financiers/]

Reviewer: ```suggestion
[The Cantillon Effect: Because of Inflation, We’re Financing the Financiers](https://fee.org/articles/the-cantillon-effect-because-of-inflation-we-re-financing-the-financiers/)
```
